### PR TITLE
[t1251] Keep tray outside body contents

### PR DESCRIPTION
**Pull request by secretrobotron**
1. Fixed up minor CSS annoyances with previous padding attempts
2. Added padding for tray specifically
3. Transitions for tray padding

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/d9b820bf127d386/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/d9b820bf127d386/output.txt

Total script time: 0.81 mins

Server is running on http://pr327.butyr.org/templates/test.html

File: css/butter.ui.deprecated.css [hunk 1]
```diff
@@ -12,10 +12,21 @@
...
 }
 
-.butter-header-spacing {
+body.butter-header-spacing {
   padding-top: 35px;
 }
 
+body.butter-tray-spacing {
+  padding-bottom: 210px;
+  -moz-transition: padding-bottom 0.5s;
```
Comment (humphd) [inline css/butter.ui.deprecated.css:21]: Should be:

  -moz-transition: opacity 0.5s;
  -ms-transition: opacity 0.5s;
  -o-transition: opacity 0.5s;
  -webkit-transition: opacity 0.5s;
  transition: opacity 0.5s;

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/0076c398eb49114/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/0076c398eb49114/output.txt

Total script time: 0.31 mins

**Pull request merged**